ImagePicker/ImagePickerController.swift

```diff
@@ -46,8 +46,8 @@ open class ImagePickerController: UIViewController {
     /// view hierarchy. For example when there is no permissions to photo library.
     private var overlayView: UIView?
     
-    private var collectionViewDataSource = ImagePickerDataSource(assetsCacheItem: ImagePickerAssetCacheItem())
-    private var collectionViewDelegate = ImagePickerDelegate()
+    var collectionViewDataSource = ImagePickerDataSource(assetsCacheItem: ImagePickerAssetCacheItem())
+    var collectionViewDelegate = ImagePickerDelegate()
```

Comment:
this could remain private, datasource not, but delegate definitely yes

Reply:
Done